src/commands/package.ts

```diff
@@ -107,9 +108,41 @@ export default class Package extends Command {
           execa('npm', ['install', '--production', `--prefix=${outputDir}`, ...(platform() === 'win32' ? ['--force'] : [])], {
             stdio: flags.verbose ? 'inherit' : 'ignore'
           }).catch(e => this.error(e, { exit: 10 }))
+      },
+      {
+        title: 'Check the SAP Cloud SDK dependencies',
+        task: async () => {
+          const { dependencies, devDependencies } = await parsePackageJson(projectDir);
+          checkOldDependencies(dependencies);
+          checkOldDependencies(devDependencies);
+        }
       }
     ]);
 
     await tasks.run();
+    this.printSuccessMessage();
+  }
+
+  private printSuccessMessage() {
+    const warnings = getWarnings();
+    const body = [
+      '🚀 Please migrate to new packages.',
+      'Please find how to migrate here:',
+      'https://github.com/SAP/cloud-sdk/blob/master/docs/how-to-switch-to-os-cloud-sdk.md'
+    ];
+    if (warnings) {
+      if (this.hasOldSDKWarnings(warnings)) {
+        this.log(boxMessage(['⚠️ Package finished with warnings:', ...warnings, '', ...body]));
+      } else {
+        this.log(boxMessage(['⚠️ Package finished with warnings:', ...warnings]));
+      }
+    } else {
+      this.log(boxMessage(['✅ Package finished successfully.']));
+    }
+  }
+
+  private hasOldSDKWarnings(warnings: string[]) {
+    const regex = RegExp('Old SAP Cloud SDK: * is detected.');
```

Comment:
is this regex correction? Don't you need a `.*` to match characters? Perhaps you can add a test that the printSuccessMessage does the right thing depending if the warnings contain an item regarding the old sdk or not.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test
shows some example and can also be used as a playground for regex with js